deploy/quickstart.sh

```diff
@@ -5,7 +5,7 @@ EPHEMERAL_STORAGE="${EPHEMERAL_STORAGE:-false}"
 oc new-project "${OCP_PROJECT}"
 ansible-playbook \
     --extra-vars namespace="${OCP_PROJECT}" \
-    --extra-vars __local_build_enabled=false \
+    --extra-vars __local_build_enabled=true \
```

Comment:
This should actually be left as `false` because the intent here is to _not_ do builds in the quickstart. This wasn't working at all because we didn't have upstream releases, but this _might_ work again (or could work again) because I have a sort of nightly release going again.